### ISSUE TITLE
Finalize website structure for initial review

### ISSUE THREAD

**Issue by eecavanna** [microbiomedata/docs#77]
Task (in scope for this ticket):
- [x] Finalize website structure: use homepage of NMDC documentation as landing page
  - Fixed via https://github.com/microbiomedata/docs/pull/78.
- [x] Refine the file tree structure (target audience: content editors)
- [ ] Document the new file tree structure -> will open separate ticket for this
- [x] Re-activate `docs.microbiomedata.org` hostname on GitHub and Cloudflare
  - Fixed (no PR).
- [x] Notify EEF on Friday PM once I am ready for her to review the website structure and landing page

Out of scope:
- Standardizing the theme across the different sub-sites (i.e. Sphinx versus MkDocs-Material)
- Updating all the content (team members are working on this in parallel)
